dataland-frontend/tests/e2e/utils/CompanyUpload.ts

```diff
@@ -23,3 +32,26 @@ export function createCompanyAndGetId(companyName: string, sector?: string): Cyp
       return $companyID.text();
     });
 }
+
+export function generateDummyCompanyInformation(companyName: string): CompanyInformation {
+  return {
+    companyName: companyName,
+    headquarters: "Imaginary-City",
+    sector: "Imaginary-Sector",
+    identifiers: [
+      { identifierType: CompanyIdentifierIdentifierTypeEnum.PermId, identifierValue: faker.random.alphaNumeric(10) },
```

Comment:
th id is generated by faker.random.alphaNumeric(10) and yes it might be possible but the chance is very slim. I do not know what the repeater frequency of this particular algorith is but I would expect millions of possible values before repeating

Reply:
You are right in theory. But what is the alternative here?
I mean there are 360.000.000.000 possible combinations here.    This is acceptable I think.
And even if it happens once that a duplicate is generated, we can rerun the test and it will (with high probability) succeed the second time. So no huge damage even if it happens in my opinion.
What do you think?

Reply:
Agree, just wanted to mention.